### PR TITLE
Added SMTP test button in the admin gui

### PR DESCRIPTION
**Pull request by BlackDex**
- Added a test button for checking the e-mail settings.
- Fixed a bug with the _post JavaScript function:
  A function was overwriten with a variable and errors were not handled
correctly like a 500 for example.

### CODE REVIEW

File: src/static/templates/admin/page.hbs [hunk 1]
```diff
@@ -260,6 +282,15 @@
             "Error inviting user", data);
         return false;
     }
+    function smtpTest() {
+        inv = document.getElementById("smtp-test-email");
+        data = JSON.stringify({ "email": inv.value });
+        inv.value = "";
+        _post("/admin/test/smtp/",
```
Comment (jjlin) [inline src/static/templates/admin/page.hbs]: You should rebase on the latest `master`; there's a `{{urlpath}}` needed here now.

Comment (BlackDex) [reply src/static/templates/admin/page.hbs]: Meh, i thought i had done a git pull!

Comment (BlackDex) [reply src/static/templates/admin/page.hbs]: @jjlin fixed that :)

**Comment by dani-garcia** [comment]
Nice thanks!

**Pull request merged**